common/lib/xmodule/xmodule/js/main_requirejs.js

```diff
@@ -0,0 +1,8 @@
+(function(requirejs) {
+    requirejs.config({
+        paths: {
+            "moment": "xmodule/include/common_static/js/vendor/moment.min"
```

Comment:
@singingwolfboy This is actually a mystery, I tried to find out a file like this which actually loading all other things in xmodule jasmine tests but couldn't find anything.